packages/lesswrong/components/recentDiscussion/RecentDiscussionThread.tsx

```diff
@@ -134,12 +156,15 @@ const RecentDiscussionThread = ({
   expandAllThreads?: boolean,
   maxLengthWords?: number,
   smallerFonts?: boolean,
+  subforumIntroPost?: boolean,
```

Comment:
This variable name made me think we were passing in a post object. Could we rename it to imply that it's a boolean? Maybe just `isSubforumIntroPost`?